cmd/genji/shell/command.go

```diff
@@ -106,3 +107,70 @@ func runHelpCmd() error {
 
 	return nil
 }
+
+// executeInsertCommand insert json data.
+func executeInsertCommand(db *genji.DB, table string, data []byte) error {
+	q := fmt.Sprintf("INSERT INTO %s VALUES ?", table)
+	dec := json.NewDecoder(bytes.NewReader(data))
+	t, err := dec.Token()
+	if err != nil {
+		return err
+	}
+
+	d, ok := t.(json.Delim)
+	// check if the token is a json delimiter.
+	if !ok {
+		return fmt.Errorf("found %q, expected '{' or '['", d.String())
+	}
+
+	switch d {
+	case '[': // Array json stream
+		var m map[string]interface{}
+		for dec.More() {
+			err := dec.Decode(&m)
```

Comment:
All the objects will be merged into the same object, this is not what we want. We want separate objects:

```bash
 genji insert -t wesh --db wesh.db '[{"a": 1},{"a": 2}]'
```

Currently:
```sql
genji> SELECT * FROM wesh;
{
  "a": 2
}
```

Expected:
```sql
genji> SELECT * FROM wesh;
{
  "a": 1
}
{
  "a": 2
}
```